pkgs/development/python-modules/chispa/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage, fetchFromGitHub, lib, poetry, ... }:
+
+buildPythonPackage rec {
+  pname = "chispa";
+  version = "0.8.2";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    repo = "chispa";
+    owner = "MrPowers";
+    rev = "v${version}";
+    sha256 = "sha256-dKiBdJNnPAJQp4OPtUjBPMqZSLNCDhfQ6OwJyHtvTrw=";
+  };
+
+  nativeBuildInputs = [ poetry ];
```

Comment:
Can we use poetry-core?

Reply:
I get the exact problem described here: https://github.com/NixOS/nixpkgs/issues/103325

Can I keep `poetry` instead?

Reply:
Opened a PR upstream trying to fix this. 

https://github.com/MrPowers/chispa/pull/40

Reply:
You can fetch the commit as a patch.

Reply:
Already got included in an upstream release, it seems